frontend/messages/sk.json

```
@@ -0,0 +1,187 @@
+    "helpMeteogramCloudCover": "Obla�nos� v r�znych nadmorsk�ch v��kach je zn�zornen� �ed�mi p�smi: ��m �ir�� je p�s, t�m v��ia je obla�nos�. Ak p�s zaber� polovicu st�pca, znamen� to, �e obla�nos� je 50 %. Ak zaberie cel� st�pec, to znamen� obla�nos� 100 %. V�imnite si, �e obla�nos� s vysokou �rov�ou je vyzna�en� v hornej �asti diagramu (ke� sa os vertik�lnej mierky prestane zobrazova�, v na�om pr�klade bude obloha pokojn�). prv� de� zamra�en�, najm� na poludnie, na druh� de� bude kopovit� obla�nos�, tret� de� bude kopovit� obla�nos�, ale poobede pr�du cirry.",
+    "helpMeteogramIsotherm1": "V��ka izotermy 0�C je zn�zornen� symbolom",
+    "helpMeteogramIsotherm2": "�ierna �iara",
+    "helpMeteogramIsotherm3": "V tomto pr�klade za��na okolo 3500 m prv� de� a zvy�uje sa a� na {maxIso} m na tret� de�.",
+    "helpMeteogramPressure1": "Naposledy, atmosf�rick� tlak je zobrazen� pomocou",
+    "helpMeteogramPressure2": "�ervenej �iary",
+    "helpMeteogramPressure3": "Stupnica je na obr�zku vpravo. Tu osciluje okolo 1010 hPa.",
+    "helpMeteogramTemperature1": "V spodnej �asti vid�me v�voj teploty na zemi",
+    "helpMeteogramTemperature2": "�erven� �iara",
+    "helpMeteogramTemperature3": "teplota rosn�ho bodu",
+    "helpMeteogramTemperature4": "modr� �iara",
+    "helpMeteogramTemperature5": "mno�stvo zr�ok. Konvek�n� zr�ky s� zobrazen� v",
+    "helpMeteogramTemperature6": "az�rov�",
+    "helpMeteogramTemperature7": "�al�ie zr�ky v",
+    "helpMeteogramTemperature8": "modr�",
+    "helpMeteogramTemperature9": "Stupnica pre zr�ky je v�avo a pre teplotu vpravo. V pr�klade sa prv� de� o�ak�va nieko�ko milimetrov zr�ok a tret� de� bude vzduch such�� (rosn� bod teplota kles�).",
+    "helpMeteogramThunderstorm": "Pr�tomnos� obrazu s bleskom nazna�uje riziko b�rky po�as d�a. ��m v��� je obr�zok, t�m vy��ie je riziko b�rky.",
+    "helpMeteogramConclusion": "Pod�a tohto diagramu je najlep�� de� na lietanie tret� de�, aj ke� popoludn� m��e by� vietor trochu pr�li� siln�.",
+    "helpSoundingIntro1": "Sond�ne diagramy zobrazuj� v�voj teploty vzduchu s nadmorskou v��kou (viac inform�ci� o sond�nych diagramoch",
+    "helpSoundingIntro2": "tu",
+    "helpSoundingIntro3": "Tu je pr�klad:",
+    "helpSounding01": "Horizont�lna os zobrazuje teplotu, zatia� �o vertik�lna os ukazuje nadmorsk� v��ku.",
+    "helpSounding02": "riadok �plne vpravo",
+    "helpSounding03": "zobrazuje v�voj teploty vzduchu s nadmorskou v��kou. A ",
+    "helpSounding04": "tenk� �ierna �iara",
+    "helpSounding05": "znamen� stabiln� vzduchov� hmotu",
+    "helpSounding06": "oran�ov� �iara",
+    "helpSounding07": "znamen� podmiene�ne nestabiln� vzduchov� hmotu a",
+    "helpSounding08": "�lt� �iara",
+    "helpSounding09": "znamen� absol�tne nestabiln� vzduchov� hmotu (ide�lna na st�panie a zr�chlenie termiky). Na druhej strane a",
+    "helpSounding10": "fialov� �iara",
+    "helpSounding11": "ozna�uje inverzn� vrstvu.",
+    "pomocSounding12": "modr� �iara",

```

Also, this key has been changed, it should still be helpSounding12.

```suggestion
    "helpSounding12": "modr� �iara",

```

---

Oh, sorry. Once more? Or here is full script:
{
    "$schema": "https://inlang.com/schema/inlang-message-format",
    "help": "nápoveda",
    "menuDisplayOnMap": "Zobraziť na mape",
    "menuOverlay": "Prekrytie",
    "menuOverlayLegend": "Zobraziť prekrytie na mape",
    "menuWind": "Vietor",
    "menuWindLegend": "Zobraziť silu a smer vetra v určitej nadmorskej výške",
    "menuForecastData": "Údaje predpovede",
    "menuModel": "Model",
    "menuGFSLegend": "Predpoveď založená na modeli GFS",
    "menuWRFLegend": "Predpoveď na základe modelu WRF",
    "menuInitializationTime": "Čas inicializácie",
    "menuInitializationTimeLegend": "Čas inicializácie spúšťania prognózy",
    "menuZone": "Zóna",
    "menuSettings": "Nastavenia",
    "menuAbout": "O",
    "menuSupport": "Podpora Soaringmeteo",
    "menuDocuments": "Dokumenty",
    "menuSoarGFS": "soarGFS (staršie)",
    "menuSoarWRF": "soarWRF (staršie)",
    "settingsLanguage": "Jazyk",
    "settingsHelpTranslate": "Pomôžte nám s prekladom!",
    "settingsWind": "Vietor",
    "settingsWindExplanation": "V predvolenom nastavení je rýchlosť vetra indikovaná číselnou hodnotou vedľa každej šípky vetra. Ak číselné hodnoty skryjete, v meteogramoch bude rýchlosť vetra indikovaná háčikmi vetra (s presnosťou 2,5 km/ h). Viac informácií o tom, ako interpretovať háčiky, nájdete v pomocníkovi k meteogramom.",
    "settingsWindShowNumericalValues": "Zobraziť číselné hodnoty",
    "settingsWindShowNumericalValuesLegend": "Zobraziť rýchlosť vetra ako číselnú hodnotu vedľa šípky vetra",
    "settingsTime": "Čas",
    "settingsTimeExplanation": "Čas môže byť zobrazený podľa vášho časového pásma alebo v UTC.",
    "settingsTimeUserTimezone": "Zobraziť čas v mojom aktuálnom časovom pásme",
    "settingsTimeUserTimezoneLegend": "Zobraziť čas pomocou môjho aktuálneho časového pásma namiesto UTC",
    "settingsTimeUTC": "Zobraziť čas podľa UTC",
    "settingsTimeUTCLegend": "Zobraziť čas UTC namiesto použitia môjho aktuálneho časového pásma",
    "locationSummary": "Súhrn",
    "locationSummaryLegend": "Súhrn predpovede pre toto miesto",
    "locationMeteogram": "Meteogram",
    "locationMeteogramLegend": "Meteogram pre toto miesto",
    "locationSounding": "Sondovanie",
    "locationSoundingLegend": "Sondovanie pre tento čas a miesto",
    "period24HoursBefore": "24 hodín predtým",
    "periodPrevious": "Predchádzajúce predpovedané obdobie",
    "periodNext": "Ďalšie predpovedané obdobie",
    "period24HoursAfter": "24 hodín po",
    "layerCloudsAndRain": "Oblaky a dážď",
    "layerCloudsAndRainLegend": "oblaky a dážď",
    "layerCumulusDepth": "Cumulus oblaky",
    "layerCumulusDepthLegend": "hĺbka kopovitých oblakov",
    "layerSoaringLayerDepth": "Hĺbka hraničnej vrstvy",
    "layerSoaringLayerDepthLegend": "hĺbka hraničnej vrstvy",
    "layerThermalVelocity": "Tepelná rýchlosť",
    "layerThermalVelocityLegend": "rýchlosť tepelného vzostupného prúdu",
    "layerThQ": "Termická kvalita",
    "layerThQLegend": "termická kvalita",
    "layerWindBoundaryLayer": "Hraničná vrstva",
    "layerWindBoundaryLayerLegend": "priemerná rýchlosť a smer vetra v hraničnej vrstve",
    "layerWindSurface": "Povrch",
    "layerWindSurfaceLegend": "rýchlosť a smer vetra na zemi",
    "layerWindSoaringLayerTop": "Horná hraničná vrstva",
    "layerWindSoaringLayerTopLegend": "rýchlosť a smer vetra v hornej časti hraničnej vrstvy",
    "layerWind300MAGL": "300 m AGL",
    "layerWind300MAGLLegend": "rýchlosť a smer vetra vo výške 300 m nad zemou",
    "layerWind2000AMSL": "2000 m AMSL",
    "layerWind2000AMSLLegend": "rýchlosť a smer vetra vo výške 2000 m nad strednou hladinou mora",
    "layerWind3000AMSL": "3000 m AMSL",
    "layerWind3000AMSLLegend": "rýchlosť a smer vetra vo výške 3000 m nad strednou hladinou mora",
    "layerWind4000AMSL": "4000 m AMSL",
    "layerWind4000AMSLLegend": "rýchlosť a smer vetra vo výške 4000 m nad strednou hladinou mora",
    "summaryTotalCloudCover": "Celková oblačnosť",
    "summaryRainfall": "Dážď",
    "summaryCumuliDepth": "Výška kumulov",
    "summarySoaringLayerDepth": "Hĺbka hraničnej vrstvy",
    "summaryThermalVelocity": "Tepelná rýchlosť",
    "summaryWeekOverview": "Prehľad týždňa",
    "summaryDayOverview": "Prehľad dňa",
    "summaryThQ": "Termická kvalita",
    "summaryWindSpeed": "Rýchlosť vetra",
    "zoneAlpsOverview": "Prehľad Álp (6 km)",
    "zoneCentralAlps": "Stredné Alpy (2 km)",
    "zoneSouthernAlps": "Južné Alpy (2 km)",
    "zoneEasternAlps": "Východné Alpy (2 km)",
    "zoneEurope": "Európa",
    "zoneAustralia": "Austrália",
    "zoneHimalaya": "Himalaya",
    "zoneNorthAmerica": "Severná Amerika",
    "zoneSouthAfrica": "Južná Afrika",
    "zoneSouthAmerica": "Južná Amerika",
    "helpIntro1": "Soaringmeteo je bezplatná webová stránka s predpoveďou počasia vyvinutá vášnivými pilotmi. Zvážte vytvorenie",
    "helpIntro2": "daru ",
    "helpIntro3": "a tak nám pomôcť pokryť naše náklady.",
    "helpCurrentForecast": "To, čo vidíte, je predpoveď počasia na {forecastDate} z modelu {model} inicializovaného o {initializationTime}.",
    "helpWrfModel1": "Prevádzkujeme",
    "helpWrfModel2": "model WRF",
    "helpWrfModel3": "na našich vlastných serveroch a nakonfigurovali sme ho nad alpským regiónom. Výsledky sa zverejňujú každý deň okolo 05:00, 11:00, 17:00 a 23:00 SELČ.",
    "helpGfsModel1": "Výsledky",
    "helpGfsModel2": "model GFS",
    "helpGfsModel3": "poskytuje spoločnosť",
    "helpGfsModel4": "Výsledky sa zverejňujú každý deň okolo 07:00 a 19:00 SELČ.",
    "helpBurgerMenu": "Pomocou ľavého horného menu vyberte, ktoré informácie sa majú zobraziť na mape (teplotná kvalita, termická rýchlosť, rýchlosť a smer vetra atď.). Môžete tiež vybrať iný model predpovede počasia alebo inú oblasť sveta (tip: po výbere obľúbeného modelu a geografickej zóny si stránku uložte do záložiek).",
    "helpOverlay": "Aktuálne, vidíte",
    "helpWindLayer": "Tiež vidíš",
    "helpBottom1": "Kliknutím na mapu zobrazíte meteogramy a zvukové diagramy pre toto miesto.",
    "helpBottom2": "Kedykoľvek znova kliknite na tlačidlo pomocníka, aby ste získali vysvetlenie toho, čo aktuálne vidíte.",
    "helpFeedback1": "Niečo nefunguje podľa očakávania? Vyplňte prosím",
    "helpFeedback2": "V prípade ďalších otázok nám pošlite",
    "helpLayerThQ": "Naznačuje potenciál termického lietania, od 0 % (slabá termika alebo veľmi silný vietor) do 100 % (silná, vysoká termika, slabý vietor). Hľadajte biele alebo modré plochy (celá farebná škála je zobrazená vpravo dole na obrazovke). Termická kvalita zohľadňuje hĺbku stúpajúcej vrstvy, otepľovanie zeme a priemernú rýchlosť vetra v rámci hraničnej vrstvy tohto ukazovateľa.",
    "helpLayerWind": "Smer vetra je znázornený šípkou. Vietor prúdi v smere šípky. Napríklad šípka, ktorá ukazuje doprava znamená, že vietor prichádza zo západu a smeruje na východ. Rýchlosť vetra je zobraziť v km/h vedľa šípky Rýchlosť vetra môžete skryť v Nastaveniach (z hlavného menu).",
    "helpLayerCloudsAndRain": "Označuje oblačnosť, ako aj množstvo zrážok v danom časovom období. Oblačnosť sa zobrazuje s bielymi a sivými odtieňmi a zrážky v milimetroch sa zobrazujú farbami, ako je uvedené v legende mapy." ,
    "helpLayerCumuliDepth1": "Komulové oblaky sú oblaky spôsobené tepelnou aktivitou. Žiadne kopovité oblaky neznamená žiadne stúpanie alebo modré stúpanie. Hlboké kopovité oblaky znamenajú riziko nadmerného rozvoja.",
    "helpLayerCumuliDepth2": "Farebná škála je zobrazená v pravom dolnom rohu obrazovky.",
    "helpLayerSoaringLayerDepth1": "Hraničná vrstva",
    "helpLayerSoaringLayerDepth2": "je oblasť atmosféry, v ktorej môžeme očakávať stúpanie a stúpanie. Hĺbka hraničnej vrstvy nám hovorí, ako vysoko môžeme stúpať. Napríklad hodnota 850 m znamená, že môžeme stúpať až 850 m nad úrovňou terénu sa uprednostňujú hodnoty vyššie ako 750 m.",
    "helpLayerSoaringLayerDepth3": "V prítomnosti kopovitých mrakov obmedzíme hĺbku hraničnej vrstvy na základňu mrakov. Ak je napríklad hĺbka hraničnej vrstvy 1 500 m, ale základňa kopovitého oblaku je vo výške 1 200 m nad úrovňou zeme, ukážeme hĺbka hraničnej vrstvy 1200 m.",
    "helpLayerSoaringLayerDepth4": "Farebná škála je zobrazená v pravom dolnom rohu obrazovky.",
    "helpLayerThermalVelocity": "Rýchlosť tepelného vzostupného prúdu sa odhaduje z hĺbky hraničnej vrstvy a otepľovania zeme. Farebná škála je zobrazená v pravej dolnej časti obrazovky.",
    "helpMeteogramIntro": "Meteogramy zobrazujú predpoveď počasia pre zvolené miesto v priebehu času. Tu je príklad trojdňového meteogramu, ktorý sme vytvorili na účely dokumentácie:",
    "helpMeteogramTopRow1": "Ten ",
    "helpMeteogramTopRow2": "horný riadok ",
    "helpMeteogramTopRow3": "('ThQ') zobrazuje odhadovanú tepelnú kvalitu (medzi 0 % a 100 %). Čím vyššie číslo, tým väčšia šanca na preletenie termiky. Zohľadňuje hĺbku hraničnej vrstvy, priemernú termickú rýchlosť , rýchlosť vetra a otepľovanie zeme Ak sa chcete dozvedieť viac o tom, ako to funguje, vyberte v zobrazení mapy vrstvu „Kvalita tepla.",
    "helpMeteogramScndRow1": "",
    "helpMeteogramScndRow2": "Druhý riadok ",
    "helpMeteogramScndRow3": "('m/s') zobrazuje odhadovanú priemernú termickú rýchlosť (v m/s) v rámci hraničnej vrstvy. Hodnoty nad 1 m/s zvyčajne znamenajú, že termika by mala byť dostatočne silná na to, aby zostala vo vzduchu. Hodnoty nad 2 m/s znamenajú dobré stúpanie.",
    "helpMeteogramAirgram": "Airgram pod týmito číslami zobrazuje rôzne vlastnosti vzduchu na vybranom mieste v priebehu času. Stupnica vľavo ukazuje nadmorskú výšku. V tomto príklade začína na {groundLevel} m, čo je nadmorská výška vybratého miesta podľa aktuálneho predpovedného modelu.",
    "helpMeteogramBoundaryLayer1": "",
    "helpMeteogramBoundaryLayer2": "Zelená plocha ",
    "helpMeteogramBoundaryLayer3": "ukazuje ",
    "helpMeteogramBoundaryLayer4": "planetárnu hraničnú vrstvu, ",
    "helpMeteogramBoundaryLayer5": "čo je časť atmosféry, kde môžeme očakávať, že nájdeme termiku a stúpame. V tomto príklade vidíme, že hraničná vrstva dosiahne {boundaryLayerDepth} m v strede posledného dňa. Na lietanie cross-country je dobré mať hraničnú vrstvu vo výške aspoň 750 m nad úrovňou terénu.",
    "helpMeteogramInversion1": "",
    "helpMeteogramInversion2": "Fialové prúžky ",
    "helpMeteogramInversion3": "označujú inverzné vrstvy. Inverzie sú časti atmosféry, kde je vzduch veľmi stabilný. Blokujú termiku a vývoj konvekčných oblakov.",
    "helpMeteogramAir1": "",
    "helpMeteogramAir2": "Vietor ",
    "helpMeteogramAir3": "a ",
    "helpMeteogramAir4": "oblaky ",
    "helpMeteogramAir5": "sú na tomto diagrame zobrazené aj v rôznych nadmorských výškach. Napríklad v hraničnej vrstve je prvé dva dni mierny vietor (medzi 15 km/h a 30 km/h) a tretí slabý vietor deň (5 až 15 km/h vietor fúka druhý deň od juhu.",
    "helpMeteogramWindBarbs": "Vietor je znázornený veternými háčikmi. Počet 'vlajok' v šípke označuje rýchlosť s presnosťou 2,5 km/h. Napríklad šípka s jednou krátkou vlajkou označuje rýchlosť medzi 0 a 2,5 km/h Ak má dve krátke vlajky, znamená to rýchlosť medzi 2,5 a 5 km/h Dve dlhé vlajky znamenajú rýchlosť medzi 7,5 a 10 km/h 20 km/h atď. Prípadne môžete rýchlosť vetra zobraziť ako číselnú hodnotu tak, že prejdete do Nastavenia z ľavého horného menu.",
    "helpMeteogramCumulus1": "Oblaky kopovité sú zobrazené pomocou",
    "helpMeteogramCumulus2": "obrázok bieleho oblaku",
    "helpMeteogramCumulus3": "Piloti klzákov nemôžu lietať vyššie, ako je základňa mrakov. Keď sa na oblakoch nezobrazuje vôbec žiadny obrázok, znamená to, že budú „modré stúpačky“. Prítomnosť kopovitých oblakov je dobrá vec na prelietavanie terénom, ale ak sa vyvinú príliš vysoko, môžu spôsobiť prehánky alebo búrky.",
    "helpMeteogramCloudCover": "Oblačnosť v rôznych nadmorských výškach je znázornená šedými pásmi: čím širší je pás, tým väčšia je oblačnosť. Ak pás zaberá polovicu stĺpca, znamená to, že oblačnosť je 50 %. Ak zaberie celý stĺpec, to znamená oblačnosť 100 %. Všimnite si, že oblačnosť s vysokou úrovňou je vyznačená v hornej časti diagramu (keď sa os vertikálnej mierky prestane zobrazovať, v našom príklade bude obloha pokojná). prvý deň zamračené, najmä na poludnie, na druhý deň bude kopovitá oblačnosť, tretí deň bude kopovitá oblačnosť, ale poobede prídu cirry.",
    "helpMeteogramIsotherm1": "Výška izotermy 0°C je znázornená symbolom",
    "helpMeteogramIsotherm2": "čierna čiara",
    "helpMeteogramIsotherm3": "V tomto príklade začína okolo 3500 m prvý deň a zvyšuje sa až na {maxIso} m na tretí deň.",
    "helpMeteogramPressure1": "Naposledy, atmosférický tlak je zobrazený pomocou",
    "helpMeteogramPressure2": "červenej čiary",
    "helpMeteogramPressure3": "Stupnica je na obrázku vpravo. Tu osciluje okolo 1010 hPa.",
    "helpMeteogramTemperature1": "V spodnej časti vidíme vývoj teploty na zemi",
    "helpMeteogramTemperature2": "červená čiara",
    "helpMeteogramTemperature3": "teplota rosného bodu",
    "helpMeteogramTemperature4": "modrá čiara",
    "helpMeteogramTemperature5": "množstvo zrážok. Konvekčné zrážky sú zobrazené v",
    "helpMeteogramTemperature6": "azúrová",
    "helpMeteogramTemperature7": "ďalšie zrážky v",
    "helpMeteogramTemperature8": "modrá",
    "helpMeteogramTemperature9": "Stupnica pre zrážky je vľavo a pre teplotu vpravo. V príklade sa prvý deň očakáva niekoľko milimetrov zrážok a tretí deň bude vzduch suchší (rosný bod teplota klesá).",
    "helpMeteogramThunderstorm": "Prítomnosť obrazu s bleskom naznačuje riziko búrky počas dňa. Čím väčší je obrázok, tým vyššie je riziko búrky.",
    "helpMeteogramConclusion": "Podľa tohto diagramu je najlepší deň na lietanie tretí deň, aj keď popoludní môže byť vietor trochu príliš silný.",
    "helpSoundingIntro1": "Sondážne diagramy zobrazujú vývoj teploty vzduchu s nadmorskou výškou (viac informácií o sondážnych diagramoch",
    "helpSoundingIntro2": "tu",
    "helpSoundingIntro3": "Tu je príklad:",
    "helpSounding01": "Horizontálna os zobrazuje teplotu, zatiaľ čo vertikálna os ukazuje nadmorskú výšku.",
    "helpSounding02": "riadok úplne vpravo",
    "helpSounding03": "zobrazuje vývoj teploty vzduchu s nadmorskou výškou. A ",
    "helpSounding04": "tenká čierna čiara",
    "helpSounding05": "znamená stabilnú vzduchovú hmotu",
    "helpSounding06": "oranžová čiara",
    "helpSounding07": "znamená podmienečne nestabilnú vzduchovú hmotu a",
    "helpSounding08": "žltá čiara",
    "helpSounding09": "znamená absolútne nestabilnú vzduchovú hmotu (ideálna na stúpanie a zrýchlenie termiky). Na druhej strane a",
    "helpSounding10": "fialová čiara",
    "helpSounding11": "označuje inverznú vrstvu.",
    "pomocSounding12": "modrá čiara",
    "helpSounding13": "zobrazuje vývoj teploty rosného bodu s nadmorskou výškou.",
    "helpSoundingBoundaryLayer1": "The",
    "helpSoundingBoundaryLayer2": "zelená plocha",
    "helpSoundingBoundaryLayer3": "zobrazuje planetárnu hraničnú vrstvu, čo je časť atmosféry, kde môžeme očakávať, že nájdeme termiku a stúpame.",
    "helpSoundingCloudCover1": "Vpravo",
    "helpSoundingCloudCover2": "biele alebo sivé pásy",
    "helpSoundingCloudCover3": "označte oblačnosť v rôznych nadmorských výškach. Čím širší je pás, tým väčšia je oblačnosť. Keď je pás veľký ako pravý stĺpec, znamená to 100% oblačnosť.",
    "helpSoundingCumulus1": "Prítomnosť kopovitých oblakov je znázornená a",
    "helpSoundingCumulus2": "obrázok bieleho oblaku",
    "helpSoundingCumulus3": "Výška základne kopovitých oblakov je tiež napísaná pod obrázkom vpravo na diagrame.",
    "helpSoundingWind": "Vľavo je rýchlosť a smer vetra zobrazená v rôznych nadmorských výškach pomocou veterných ostňov.",
    "helpSoundingThermalVelocity": "Text „2,1 m/s“ (v strede stúpajúcej vrstvy, vpravo na diagrame) vám hovorí o odhadovanej priemernej tepelnej rýchlosti v rámci hraničnej vrstvy. Hodnoty nad 1 m/s zvyčajne znamenajú, že stúpanie by malo byť dostatočne silné na to, aby sa udržalo vo vzduchu, hodnoty nad 2 m/s znamenajú dobré stúpanie.",
    "helpSoundingExpand": "V predvolenom nastavení diagram zobrazuje iba vzduchovú hmotu v hraničnej vrstve a niekoľko tisíc metrov nad ňou. Kliknutím na tlačidlo v pravom hornom rohu môžete diagram rozbaliť a zobraziť celú troposféru."
   }